lib/worker.js

```diff
@@ -121,6 +167,8 @@ Worker.prototype.run = function () {
     queue: this.queue,
     job: this.job
   }
+  this._incMonitor('ponos')
+  var timer = monitor.timer('ponos.timer', !process.env.WORKER_MONITOR_DISABLED, this._eventTags())
```

Comment:
good questions @bkendall. So second parameter to `monitor.timer` is `start`. If it false timer will not be started.
`timer.stop` will also work correctly.
Using this flag I don't need to add additional `if` around `monitor.timer`.